### ISSUE TITLE
How can we prevent elm-live from monitoring `node_modules/`?

### ISSUE THREAD

**Issue by Natim** [tomekwi/elm-live#103]
```
> elm-live Main.elm --output=hoverpad.js --open

Success! Compiled 1 module.                                         
Successfully generated app.js
elm-live:
  You’ve added `node_modules/.bin/strip-indent`. Rebuilding!
elm-live:
  You’ve added `node_modules/watchify/node_modules/.bin/browserify`. Rebuilding!
elm-live:
  You’ve added `node_modules/npm/node_modules/request/node_modules/.bin/har-validator`. Rebuilding!
elm-live:
  You’ve added `node_modules/npm/node_modules/request/node_modules/http-signature/node_modules/.bin/sshpk-conv`. Rebuilding!
```

**Comment by Natim** [comment]
Refs https://github.com/tomekwi/elm-live/pull/90

**Comment by tomekwi** [comment]
Hey, thanks for opening 🙂 Can you please post more info about the version of elm-live and the OS you’re running?

Also, please read through https://github.com/tomekwi/elm-live/issues/97 for more context about picking up changes from `node_modules`. (TLDR: This is not intended and we’re still trying to put the [short, self-contained, compilable example](http://sscce.org/). Your help would be appreciated.)

I’m keeping this issue open though, because #97 seems to involve two cross-cutting concerns – and picking up `node_modules` is only one of them.

**Comment by Natim** [comment]
I have no idea how to write such a test. I am bit puzzle because you seem to be using `**/*.{html,css,js}` and `**/*.elm` so I don't see why this command are monitored to be honest.

I guess a way to reproduce the test is to run elm-live in a thread and touching some files in a fake node_module/.bin repository in the meantime and see that elm-live doesn't rebuild when we do.

**Comment by Natim** [comment]
After investigation it seems that the problem is coming from chokidar.
https://github.com/paulmillr/chokidar/issues/544

**Comment by Natim** [comment]
Any help on fixing this, it is really a pain while using Elm-live on a daily basis

**Comment by tomekwi** [comment]
Fixed in v2.7.5. Really sorry for the long wait on this, it was a crazy couple of months for me! All getting slightly better now though 😄
